src/datetime/tests.rs

```diff
@@ -469,7 +469,7 @@ fn test_rfc3339_opts() {
     assert_eq!(dt.to_rfc3339_opts(Nanos, false), "2018-01-11T10:05:13.084660000+08:00");
     assert_eq!(dt.to_rfc3339_opts(AutoSi, false), "2018-01-11T10:05:13.084660+08:00");
 
-    let ut = DateTime::<Utc>::from_utc(dt.naive_utc(), Utc);
+    let ut = dt.naive_utc().and_utc();
```

Comment:
The old behavior is deprecated but still supported.  Can this test case code be modified to test the deprecated behavior and the new behavior (and all other test case code in this PR)?

Reply:
Hmm. We had three tests that happened to use it, but not in a meaning ful way.

Do we want to add new tests for deprecated methods?

Reply:
> Hmm. We had three tests that happened to use it, but not in a meaningful way.
> 
> Do we want to add new tests for deprecated methods?

I think it is worthwhile to still test formally deprecated code.

Reply:
I added a basic test.